orchestra/communication/staffing.py

```diff
@@ -117,19 +120,58 @@ def send_staffing_requests(
                 Q(last_inquiry_sent__lte=cutoff_datetime)))
 
     for request in requests:
-        send_request_inquiries(staffbot, request, worker_batch_size)
+        send_request_inquiries(staffbot, request)
+
+
+def get_worker_priorities(workers_with_inquiries):
+    def get_workers_in_priority_order():
+        return (Worker.objects
+                .exclude(id__in=workers_with_inquiries)
+                .order_by('-staffing_priority', '?'))
+
+    # Ping 2 workers in priority order every 2 minutes
+    return [(get_workers_in_priority_order, 2, 2)]
```

Comment:
these numbers are based on the appropriate `settings` values (`settings.ORCHESTRA_STAFFBOT_WORKER_BATCH_SIZE`, `settings.ORCHESTRA_STAFFBOT_BATCH_FREQUENCY`)